lib/public_key/src/pubkey_cert.erl

```diff
@@ -1864,8 +1874,8 @@ missing_basic_constraints(OtpCert, SelfSigned, ValidationState, VerifyFun, UserS
 	     UserState}
     end.
 
-is_valid_key_usage(KeyUse, Use) ->
-    lists:member(Use, KeyUse).
+is_valid_key_usage(KeyUses, Use) ->
```

Comment:
I'm not sure about value added by this function.
Also it is unfortunate it swaps the order of arguments. confusing and not optimal as I've read in some of Raimo comments.

Reply:
Ah , well lets remove it then.